avatars/animationHelpers.js

```diff
@@ -72,6 +72,7 @@ let fallLoop;
 // let swordSideSlash;
 // let swordTopDownSlash;
 let hurtAnimations;
+let lastF;
```

Comment:
This is a new global variable. How does this work when many avatars are animating?

Reply:
Reverted fixing of unuseAnimation not stop problem, https://github.com/webaverse/app/pull/2804/commits/29b8e11ef0d51b86f7008a50a6c835a694263148
because it's not very related to this PR and not a serious problem,
should try in another PR afterwards.

Reply:
Made it as local variable instead of global. https://github.com/webaverse/app/pull/2804/commits/0ccd4c2681f2b379ce1782b3ee7b01d20f4550e4